src/ios/CDVWKWebViewEngine.m

```diff
@@ -431,6 +444,37 @@ - (void)onAppWillEnterForeground:(NSNotification *)notification {
     }
 }
 
+-(void)keyboardWillHide
+{
+    if (@available(iOS 12.0, *)) {
+        timer = [NSTimer scheduledTimerWithTimeInterval:0 target:self selector:@selector(keyboardDisplacementFix) userInfo:nil repeats:false];
+        [[NSRunLoop mainRunLoop] addTimer:timer forMode:NSRunLoopCommonModes];
+    }
+}
+
+-(void)keyboardWillShow
+{
+    if (timer != nil) {
+        [timer invalidate];
+    }
+}
+
+-(void)keyboardDisplacementFix
+{
+    WKWebView *webview = (WKWebView*) self.webView;
+    for (UIView* v in webview.subviews ){
```

Comment:
Rather than iterating through the webview's subviews, you can just access [`webview.scrollView`](https://developer.apple.com/documentation/webkit/wkwebview/1614784-scrollview?language=objc).

Reply:
Simplified. Thanks @ravron